### ISSUE TITLE
Replace Module Info Dummies with Real Data

### ISSUE THREAD

**Issue by jdrueckert** [MovingBlocks/movingblocks.github.com#43]
@majordwarf Can you please add some information here on the original approach to gather module info?

Potentially relevant for future work on this: https://github.com/MovingBlocks/Terasology/pull/3966

**Comment by ryuk156** [comment]
@majordwarf can you please explain about Modules and how we are going to implement that

**Comment by majordwarf** [comment]
Well, the data present in the repository aside from the `modules/` directory will be changed/modified manually. To gather the modules' info, I have had two approaches. One would be a single-time pipeline to scrape data of all of the modules together and one to be a part of the module build pipeline to update the earlier gathered data on every `push` event or module build trigger. 

We'd typically look for data in `module.txt`, `README.md`, module logo, and cover image. If the logo or cover images are not found we revert back to a boilerplate.

The single-time pipeline I wrote in 2019 resides here: [https://github.com/dwarf-test/master-scrape](https://github.com/dwarf-test/master-scrape)

The scraped data is stored in a separate repo, during the module site deployment, the data is fetched from this repo and formatted as per the need of the Module site. The temporary data in `modules/` would be purged first before we start to generate actual module data.

**Comment by jdrueckert** [comment]
done
